src/index.spec.ts

```diff
@@ -238,6 +238,73 @@ describe("calculateDepreciation()", () => {
     }])
   });
 
+  it("should not return endAmount of 1 for small amount like 31", () => {
```

Comment:
ok, I change that.

Reply:
We now have:

- should return endAmount of 0
- should return endAmount of 0 for small amount like 31 - purchase made in Jan
- should return endAmount of 0 for small amount like 31 - purchase made after Jan
- should not return endAmount of -1
- should not return endAmount of 3
- should not return endAmount of 5

I think this is becoming confusing. Perhaps we should focus on the semantics instead of on the pure `endAmount`, for example, “should depreciate rounding errors in the last year” or “should handle remainder”.

In any case, I think it’s fine to stick to the existing convention in this pull request.